tools/fuchsia/sdk/sdk_targets.gni

```diff
@@ -107,16 +107,19 @@ template("sdk_targets") {
     part_meta_rebased = "$fuchsia_sdk_path/$part_meta"
 
     part_meta_json = read_file(part_meta_rebased, "json")
-    subtarget_name = part_meta_json.name
 
-    if (part.type == target_type) {
-      if (target_type == "dart_library") {
-        _fuchsia_dart_library(subtarget_name) {
-          meta = part_meta_json
-        }
-      } else if (target_type == "fidl_library") {
-        _fuchsia_fidl_library(subtarget_name) {
-          meta = part_meta_json
+    if (defined(part_meta_json.name)) {
```

Comment:
Why not check for:

if (part_meta != "version_history.json") {

here?  Checking for the presence of the "name" field is really overly broad and might hide other bugs in the future.

Reply:
That is a good idea. I will need to update the buildroot change as well to do the same